auth/src/main/res/layout/register_email_layout.xml

```diff
@@ -66,7 +66,7 @@
             style="@style/FirebaseUI.Text.BodyText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="336dp"
+            android:layout_marginTop="@dimen/register_field_padding_vert_3x"
```

Comment:
And I found what we need:
https://developer.android.com/reference/android/support/design/widget/TextInputLayout.html#attr_android.support.design:errorEnabled

Woot!